assets/js/main.js

```diff
@@ -25,6 +25,25 @@ require('./modules/logos').init();
 require('./modules/materials').init();
 require('./modules/forms').init();
 
+console.log(window.AppConfig);
+if (window.AppConfig.adminEndpoint) {
+    fetch(window.AppConfig.adminEndpoint, { credentials: 'include' })
```

Comment:
Worth checking for `fetch` support here (eg. not in IE11) before making this call?